html/about.html

```diff
@@ -101,13 +118,15 @@ <h4>
 					Take a look to the first two trailers of the movies, wich was filmed
 					in New Zeland
 				</h4>
-				<div class="fellowship-container">
-					<h3>2001</h3>
-					<h4>Fellowship of the Ring</h4>
-				</div>
-				<div class="two-thrones-container">
-					<h3>2002</h3>
-					<h4>The Two Thrones</h4>
+				<div class="trailers-container">
+					<div class="fellowship-container">
+						<h3>2001</h3>
+						<h4>Fellowship of the Ring</h4>
+					</div>
+					<div class="two-thrones-container">
+						<h3>2002</h3>
+						<h4>The Two Thrones</h4>
+					</div>
 				</div>
 			</article>
 		</section>
```

Comment:
- [ ] As the previous reviewer pointed out please fix the footer of the about page according to the design.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/169388674-b88eef14-3393-4fec-9b89-99cdf89b5e63.png)

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/169388786-c11738e3-77a4-45d1-a402-ba977b8d5200.png)